src/scrollspy/scrollspy.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  iit,
+  it,
+  ddescribe,
+  describe,
+  expect,
+  inject,
+  async,
+  beforeEach,
+  beforeEachProviders
+} from '@angular/core/testing';
+
+import {TestComponentBuilder} from '@angular/compiler/testing';
+
+import {Component, provide} from '@angular/core';
+
+import {NgbScrollSpier, NgbScrollSpy, NgbScrollTarget} from './scrollspy';
+
+function expectToBeActive(componentElement, idx) {
+  let anchors = componentElement.querySelectorAll('a');
+  Array.from(anchors).forEach((anchor, aIdx) => {
+    if (aIdx === idx) {
+      expect(anchor).toHaveCssClass('active');
+    } else {
+      expect(anchor).not.toHaveCssClass('active');
+    }
+  });
+}
+
+describe('ngb-scroll-spy', () => {
+  let html: string;
+
+  beforeEach(() => {
+    html = `
+      <div>
+        <navbar>
+          <nav>
+            <ul>
+              <li><a href [ngbScrollSpy]="selectors[0]">Anchor 1</a></li>
+              <li><a href [ngbScrollSpy]="selectors[1]">Anchor 2</a></li>
+              <li><a href [ngbScrollSpy]="selectors[2]">Anchor 3</a></li>
+            </ul>
+          </nav>
+        </navbar>
+        <section ngbScrollTarget>
+          <div id="foo" style="height: 500px; width: 100px"></div>
+          <div id="bar" style="height: 500px; width: 100px"></div>
+          <div id="baz" style="height: 500px; width: 100px"></div>
+        </section>
+      </div>
+    `;
+  });
+
+  beforeEachProviders(() => [provide(NgbScrollSpier, {useClass: NgbScrollSpier})]);
+
+  it('should have no active class', async(inject([TestComponentBuilder], (tcb) => {
```

Comment:
Test name should say under which conditions there should be no active class. The way it reads now (`'should have no active class'`) makes me think that it should never have an active class. Are you trying to say something like `'should have no active class when no elements spied on are visible'`?